src/neo/SmartContract/Native/GasToken.cs

```diff
@@ -28,7 +28,7 @@ internal override void OnPersist(ApplicationEngine engine)
             }
             ECPoint[] validators = NEO.GetNextBlockValidators(engine.Snapshot, engine.ProtocolSettings.ValidatorsCount);
             UInt160 primary = Contract.CreateSignatureRedeemScript(validators[engine.PersistingBlock.PrimaryIndex]).ToScriptHash();
-            Mint(engine, primary, totalNetworkFee, false);
+            _ = Mint(engine, primary, totalNetworkFee, false);
```

Comment:
this call should be awaited. Since it's void return, we need a try/catch block as well